docs/Microservices/overview.md

```diff
@@ -84,6 +84,28 @@ For more information about power states supported by the REST APIs, see [Intel®
 
 Intel® AMT enables remote management of a device, even when the OS isn't running, through KVM over IP support. No additional equipment is needed for this feature.  With KVM control, IT administrators can access and update PCs and devices as if they were onsite. It eliminates the need for remote KVM switches and other hardware. 
 
+## Passwords
+
+There are four levels of passwords: 
+
+1. **Intel® Manageability Engine BIOS Extensions (MEBX) Password:** Intel® MEBX enables the configuration of the Intel Manageability Engine (ME) platform. The Intel® MEBX password along with other configuration details is set on the managed device in the BIOS. For details about entering the Intel® MEBX BIOS settings, see [MEBX Unprovision](../Topics/MEBX/unprovision.md).
+
+2. **Sample Web UI Password:** The system administrator uses this password on the development system when logging into the Sample Web UI. To login, see [Login to Sample Web UI](../General/loginToRPS.md)
```

Comment:
This is only used when MPS is issuing the JWT.  It like likely that in a production environment, this will be disabled and the ISV will use their User Authentication methods for authenticating users and issuing their own Auth Token that Kong validates.  I would modify this to be something like the following:
"By default, the Sample Web UI Password uses the default MPS user authentication credential to have MPS issue a JWT.  In most production environments, this is replaced by OAuth2, OpenID, or another user authentication service to issue an Auth Token that can be validated by the API Gateway."